packages/compartment-mapper/test/fixtures-0/node_modules/typecommon/main.js

```diff
@@ -3,4 +3,4 @@ const mjs = require('./mjs.mjs');
 const cjs = require('./cjs.cjs');
 const json = require('./json.json');
 
-module.exports = [js, mjs, cjs, json];
+module.exports = [js, mjs, cjs.default, json];
```

Comment:
Ah, the old test case was definitely invalid. CommonJS importing CommonJS should not treat `default` as special! Great change.

Do we also test `module.exports = 42`? Changing `cjs.cjs` to do this instead might be more in the spirit of this test, but also I want to capture the case of a CommonJS uses `default` as an exported property name like you do here.

Reply:
I just wanted to make it pass. I'll do both to cover more regression possibilities